src/gollum/sonora.py

```diff
@@ -123,17 +123,111 @@ def __init__(
         else:
             super().__init__(*args, **kwargs)
 
+class Sonora2021Spectrum(PrecomputedSpectrum):
+    r"""
+    A container for a single Sonora precomputed synthetic spectrum of a brown dwarfs or free-floating
+    Gas Giant planet.
+
+    Args:
+        Teff (int): The Teff label of the Sonora model to read in.  Must be on the Sonora grid.
+        logg (float): The logg label of the Sonora model to read in.  Must be on the Sonora grid.
+        path (str): The path to your local Sonora grid library.  You must have the Sonora grid downloaded locally.  Default: "~/libraries/raw/Sonora/"
+        wl_lo (float): the bluest wavelength of the models to keep (Angstroms)
+        wl_hi (float): the reddest wavelength of the models to keep (Angstroms)
+        """
+
+    def __init__(
+        self, *args, teff=None, logg=None, met = None, path=None, wl_lo=8038, wl_hi=12849, **kwargs
```

Comment:
I recommend spelling out `metallicity` rather than abbreviating.  Or maybe we could do `Z`.  We can't use special characters like `[Fe/H]`, so maybe `Fe_H` could work.  The only problem is that `FeH` means Iron Hydride. So for those reasons I think `metallicity` is probably best.